src/app/users/users.component.ts

```diff
@@ -15,6 +17,7 @@ export class UsersComponent implements OnInit {
   isLoading = false;
```

Comment:
add also `isError` or something similar and display an appropriate message on UI